charts/trento-server/charts/trento-wanda/templates/configmap.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
 data:
   DATABASE_URL: "ecto://{{ .Values.global.postgresql.postgresqlUsername }}:{{ .Values.global.postgresql.postgresqlPassword }}@{{ .Release.Name }}-{{ .Values.global.postgresql.name }}/trento_wanda"
   AMQP_URL: "amqp://trento:trento@{{ .Release.Name }}-{{ .Values.global.rabbitmq.name }}:{{ .Values.global.rabbitmq.servicePort }}"
-  CORS_ORIGIN: "http://{{ .Release.Name }}-{{ .Values.global.trentoWanda.name }}/api"
+  CORS_ENABLED: "false"
```

Comment:
could we add this and the CORS_ORIGIN in the form of values?

Reply:
To be able to modify on the fly?
Ok, I will have a look, let's see how easy or valuable it is

Reply:
@fabriziosestito Done in the last commit. The default option will be `false` for cors enable.
To use it:
```
--set trento-wanda.cors.enabled=true --set trento-wanda.cors.origin=http://localhost
```
or if we want the cors origin automatically generated
```
--set trento-wanda.cors.enabled=true
```